nautobot_ssot/tables.py

```diff
@@ -104,7 +105,7 @@ class Meta(BaseTable.Meta):
         """Metaclass attributes of SyncTable."""
 
         model = Sync
-        fields = (
+        fields = (  # pylint: disable=nb-use-fields-all
```

Comment:
Any reason why we can't use `__all__` here and in the next two instances?

Reply:
I believe this pylint rule is wrong. I've tried to use `fields = "__all__"` and also `fields = ["__all__"]` and both return errors or the pylint error. Using the list version just returns the pylint error and trying to use just the string throws a TypeError:

`TypeError: DashboardTable.fields = '__all__' (type str), but type must be one of (tuple, list, set)`.

Reply:
yes - see https://github.com/nautobot/pylint-nautobot/pull/83. Not sure when the next release will include this fix.

Reply:
> yes - see [nautobot/pylint-nautobot#83](https://github.com/nautobot/pylint-nautobot/pull/83). Not sure when the next release will include this fix.

Thanks for confirming that the rule is broken @glennmatthews!